File: app/src/main/java/com/example/sasham/testproject/network/WebestApi.java
package com.example.sasham.testproject.network;

import com.example.sasham.testproject.model.Theme;

import io.reactivex.Observable;
import retrofit2.Call;
import retrofit2.http.GET;
import retrofit2.http.Path;

public interface WebestApi {

    @GET("forum/fresh/{page}/")
    Observable<ThemesWraper> themes(@Path("page") String page);
}

File: app/src/main/java/com/example/sasham/testproject/network/WebestApi.kt
package com.example.sasham.testproject.network

import com.example.sasham.testproject.model.UsersWrapper
import io.reactivex.Observable
import retrofit2.http.GET
import retrofit2.http.Path

interface WebestApi {

    @GET("v1/forum/topics/page/{page}/")
    fun themes(@Path("page") page: String): Observable<ThemesWraper>

    @GET("v1/forum/topic/{id}/")
    fun themeMessages(@Path("id") themeId: String): Observable<MessageWraper>

    @GET("v1/forum/users/")
    fun users(): Observable<UsersWrapper>
}
